### DISCUSSION TITLE
Batch Size

### DISCUSSION THREAD

**Discussion by fairdealdave** [KnollFrank/HowBadIsMyBatch#9] [Q&A]
Is there any way to determine the number of doses in each manufacturing batch?

**Comment by KnollFrank** [comment]
Hi,

no, not in general, but please have a look at the table and it's column named "Doses shipped" of  this website [Pfizer Vaccine Distribution by ZIP Code](https://knollfrank.github.io/HowBadIsMyBatch/VaccineDistributionByZipcode.html).
